app/src/main/java/com/example/music_app/Service/MusicPlayService.kt

```diff
@@ -0,0 +1,88 @@
+package com.example.music_app.Service
+
+import android.app.Service
+import android.content.ContentUris
+import android.content.Intent
+import android.media.MediaPlayer
+import android.os.Binder
+import android.os.IBinder
+import android.os.PowerManager
+import android.provider.MediaStore
+import android.util.Log
+import java.io.IOException
+
+class MusicPlayService : Service() {
+    private val mIBinder: IBinder = LocalBinder()
```

Comment:
https://kotlinlang.org/docs/reference/coding-conventions.html
em đọc thêm về convention nha
Kotlin có khả năng tự động biên dịch kiểu mạnh mẽ
e có thể viết ntn luôn 
```suggestion
    private val binder = LocalBinder()
```